types/bitcoinjs-lib/index.d.ts

```diff
@@ -232,172 +352,969 @@ export const opcodes: {
+            };
+
+        }
+
+        namespace writeUIntBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUIntLE {
+            const prototype: {
+            };
+
+        }
+
+    }
+
+
+}
+
+
+
+export namespace address {
+    function fromBase58Check(address: string): string;
+
+    function fromOutputScript(outputScript: Buffer, network: Network): string;
+
+    function toBase58Check(hash: Buffer, version: number, ...args: any[]): any;
+
+    function toOutputScript(address: string, network: Network): any;
+
+    namespace fromBase58Check {
+        const prototype: {
+        };
+
+    }
+
+    namespace fromOutputScript {
+        const prototype: {
+        };
+
+    }
+
+    namespace toBase58Check {
+        const prototype: {
+        };
+
+    }
+
+    namespace toOutputScript {
+        const prototype: {
+        };
+
+    }
+
+}
+
+export namespace bufferutils {
+    function pushDataSize(i: any): any;
+
+    function readPushDataInt(buffer: Buffer, offset: any): any;
+
+    function readUInt64LE(buffer: Buffer, offset: any): any;
+
+    function readVarInt(buffer: Buffer, offset: any): any;
+
+    function varIntBuffer(number: number, buffer: Buffer, offset: any): any;
+
+    function varIntSize(number: number): any;
+
+    function writePushDataInt(buffer: Buffer, number: number, offset: any): any;
+
+    function writeUInt64LE(buffer: Buffer, value: any, offset: any): any;
+
+    function writeVarInt(buffer: Buffer, number: number, offset: any): any;
+
+    namespace pushDataSize {
+        const prototype: {
+        };
+
+    }
+
+    namespace readPushDataInt {
+        const prototype: {
+        };
+
+    }
+
+    namespace readUInt64LE {
+        const prototype: {
+        };
+
+    }
+
+    namespace readVarInt {
+        const prototype: {
+        };
+
+    }
+
+    namespace varIntBuffer {
+        const prototype: {
+        };
+
+    }
+
+    namespace varIntSize {
+        const prototype: {
+        };
+
+    }
+
+    namespace writePushDataInt {
+        const prototype: {
+        };
+
+    }
+
+    namespace writeUInt64LE {
+        const prototype: {
+        };
+
+    }
+
+    namespace writeVarInt {
+        const prototype: {
+        };
+
+    }
+
+}
+
+export namespace crypto {
+    function hash160(buffer: Buffer): Buffer;
+
+    function hash256(buffer: Buffer): any;
+
+    function ripemd160(buffer: Buffer): any;
+
+    function sha1(buffer: Buffer): any;
+
+    function sha256(buffer: Buffer): any;
+
+    namespace hash160 {
+        const prototype: {
+        };
+
+    }
+
+    namespace hash256 {
+        const prototype: {
+        };
+
+    }
+
+    namespace ripemd160 {
+        const prototype: {
+        };
+
+    }
+
+    namespace sha1 {
+        const prototype: {
+        };
+
+    }
+
+    namespace sha256 {
+        const prototype: {
+        };
+
+    }
+
+}
+
+export namespace script {
+    const multisig: {
```

Comment:
A lot of `script` consists of re-exports from https://github.com/bitcoinjs/bitcoinjs-lib/blob/f4e3b52325c77c0b2308eb5901aaca087c781692/src/templates/index.js. If you plan to add types for these, please review and populate the correct types; otherwise, consider removing them.